apps/passport-server/test/genericIssuance.spec.ts

```diff
@@ -62,10 +62,10 @@ import {
   LemonadeUser
 } from "./lemonade/LemonadeDataMocker";
 import {
-  customTicketHandler,
+  customTicketHandler as customLemonadeTicketHandler,
   getMockLemonadeHandlers,
   loadApolloErrorMessages,
-  unregisteredUserTicketHandler
+  unregisteredUserTicketHandler as unregisteredLemonadeUserHandler
```

Comment:
did you mean to rename the imported functions in this manner?

Reply:
Hmm, no, I intended that to rename the actual function and not just the import!